lms/djangoapps/courseware/features/problems.feature

```diff
@@ -223,10 +223,9 @@ Feature: LMS.Answer problems
         Given I am viewing a "<ProblemType>" problem
         When I answer a "<ProblemType>" problem "<InitialCorrectness>ly"
         Then my "<ProblemType>" answer is marked "<InitialCorrectness>"
-        And I input an answer on a "<ProblemType>" problem "<OtherCorrectness>ly"
+        And I reset the problem
```

Comment:
@andy-armstrong did you verify that the test as written now matches the behavior in prod?